components/src/core/Drawer/DrawerHeader.tsx

```diff
@@ -213,6 +213,7 @@ const DrawerHeaderRender: React.ForwardRefRenderFunction<unknown, DrawerHeaderPr
                                     onIconClick();
                                 }}
                                 edge={'start'}
+                                size={'large'}
```

Comment:
We need `size={'large'}` which will match padding of our previous DrawerHeader.
size={'medium'} - default value
<img width="876" alt="Screenshot 2022-03-02 at 4 13 06 PM" src="https://user-images.githubusercontent.com/10433274/156346709-036c99bc-552c-423b-bd66-14c441650a68.png">

size={'large'}
<img width="869" alt="Screenshot 2022-03-02 at 4 12 47 PM" src="https://user-images.githubusercontent.com/10433274/156346771-3fe7fade-3800-427d-9cd5-5bdf627b525d.png">